crdsonnet/renderEngines/ast.libsonnet

```diff
@@ -25,23 +26,36 @@ local xtd = import 'github.com/jsonnet-libs/xtd/main.libsonnet';
   },
 
   nestInParents(name, parents, field)::
-    j.object.members([
```

Comment:
Functions in `ast.libsonnet` generally operate on a field or arrays of fields, this ensures `nestInParents` also follows that pattern. For the other render engines there is no distinction between objects and arrays of fields.